includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -2275,15 +2275,12 @@ protected function request( $params, $api, $method, $is_site_specific = true, $u
 
 			$log_request_id = uniqid();
 
-			Logger::log(
-				Logger::format_object(
-					'REQUEST_' . $log_request_id,
-					array_merge(
-						$request_args,
-						[ 'url' => $redacted_url ],
-						null !== $body ? [ 'body' => $redacted_params ] : []
-					)
-				)
+			Logger::info(
+				sprintf( '> API REQUEST (%s): %s %s', $log_request_id, $method, $redacted_url ),
```

Comment:
I can't say that that I see much of a need of the `>`. We already have caps there, I think it's fine to drop. But I'm really nit-picking here!

Reply:
Agree, after I added `WEBHOOK` and `API`, it made things clear enough.